src/containers/ViewerPanel/index.tsx

```diff
@@ -99,6 +103,7 @@ interface ViewerPanelState {
     particleTypeIds: string[];
     height: number;
     width: number;
+    movieURL: string;
```

Comment:
since this is stored in local state and only sent down to one component, why not just have it stored by that one component?

Reply:
The `onRecordedMovie` handler receives the blob of data from the viewer, so we need to do something with that blob in the component where the callback is defined and passed to the viewer.

We could pass the blob to the lower component as a prop and define the URL there, but I figured that passing the blob around, rather than just a URL, could potentially have performance implications?

Reply:
oh never mind, I see, you needed it for the onRecordMovie prop